--- a/configure
+++ b/configure
@@ -2691,11 +2691,6 @@ fi
 
 fi
 
-LDFLAGS="$LDFLAGS -L${ac_default_prefix}/lib"
-CFLAGS="$CFLAGS -I${ac_default_prefix}/include"
-CPPFLAGS="$CPPFLAGS -I${ac_default_prefix}/include"
-
-
 for ac_func in pthread_detach
 do
 as_ac_var=`echo "ac_cv_func_$ac_func" | $as_tr_sh`
@@ -4698,7 +4693,7 @@ cat >>conftest.$ac_ext <<_ACEOF
 #ifdef HAVE_LINUX_SOCKET_H
 #include <linux/socket.h>
 #endif
-
+#include <sys/socket.h>
 #include <$ac_header>
 _ACEOF
 rm -f conftest.$ac_objext
@@ -4825,6 +4820,7 @@ cat >>conftest.$ac_ext <<_ACEOF
 #ifdef HAVE_LINUX_SOCKET_H
 #include <linux/socket.h>
 #endif
+#include <sys/socket.h>
 
 #include <$ac_header>
 _ACEOF
--- a/headers.h
+++ b/headers.h
@@ -171,6 +171,7 @@
 #endif
 
 #ifdef	HAVE_LINUX_NETLINK_H
+#include <sys/socket.h>
 #include <linux/netlink.h>
 #endif
 
--- a/main.c
+++ b/main.c
@@ -26,6 +26,8 @@
  * $Id: main.c,v 1.44 2004/09/17 05:37:58 vlm Exp $
  */
 
+#include <signal.h>
+
 #include "ipcad.h"
 #include "cfgvar.h"
 #include "servers.h"
--- a/pps.c
+++ b/pps.c
@@ -26,6 +26,7 @@
  * $Id: pps.c,v 1.27 2004/09/17 05:49:55 vlm Exp $
  */
 
+#include <signal.h>
 
 #include "ipcad.h"
 #include "opt.h"
--- a/servers.c
+++ b/servers.c
@@ -26,6 +26,8 @@
  * $Id: servers.c,v 1.10 2006/04/07 18:58:02 vlm Exp $
  */
 
+#include <signal.h>
+
 #include "ipcad.h"
 #include "servers.h"
 #include "opt.h"
