--- a/setup.py
+++ b/setup.py
@@ -329,9 +329,6 @@
             self.distribution.ext_modules = exts
         else:
             h = os.path.join(get_python_inc(), 'Python.h')
-            if not os.path.exists(h):
-                raise SystemExit('Python headers are required to build '
-                                 'Mercurial but weren\'t found in %s' % h)
 
     def run(self):
         if self.distribution.pure:
