--- a/contrib/python-zstandard/setup.py
+++ b/contrib/python-zstandard/setup.py
@@ -103,6 +103,7 @@ setup(
         'Programming Language :: Python :: 3.5',
         'Programming Language :: Python :: 3.6',
         'Programming Language :: Python :: 3.7',
+        'Programming Language :: Python :: 3.8',
     ],
     keywords='zstandard zstd compression',
     packages=['zstandard'],
--- a/contrib/python-zstandard/zstd/decompress/zstd_decompress.c
+++ b/contrib/python-zstandard/zstd/decompress/zstd_decompress.c
@@ -88,6 +88,16 @@ size_t ZSTD_estimateDCtxSize(void) { ret
 
 static size_t ZSTD_startingInputLength(ZSTD_format_e format)
 {
+/* WTF? */
+#ifdef ZSTD_FRAMEHEADERSIZE_PREFIX
+#undef ZSTD_FRAMEHEADERSIZE_PREFIX
+#  define ZSTD_FRAMEHEADERSIZE_PREFIX 5
+#endif
+
+#ifdef ZSTD_FRAMEHEADERSIZE_MIN
+#undef ZSTD_FRAMEHEADERSIZE_MIN
+#  define ZSTD_FRAMEHEADERSIZE_MIN 6
+#endif
     size_t const startingInputLength = (format==ZSTD_f_zstd1_magicless) ?
                     ZSTD_FRAMEHEADERSIZE_PREFIX - ZSTD_FRAMEIDSIZE :
                     ZSTD_FRAMEHEADERSIZE_PREFIX;
