diff -Naur wview-5.21.7.orig/alarms/Makefile.am wview-5.21.7/alarms/Makefile.am
--- wview-5.21.7.orig/alarms/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/alarms/Makefile.am	2016-07-24 09:56:32.138525000 +0300
@@ -8,7 +8,6 @@
 # define include directories
 INCLUDES = \
 		-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -35,10 +34,4 @@
 # define libraries
 wvalarmd_LDADD   =
 
-# define library directories
-wvalarmd_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wvalarmd_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/cwop/Makefile.am wview-5.21.7/cwop/Makefile.am
--- wview-5.21.7.orig/cwop/Makefile.am	2014-05-28 04:22:50.000000000 +0400
+++ wview-5.21.7/cwop/Makefile.am	2016-07-24 09:55:46.111523001 +0300
@@ -8,7 +8,6 @@
 # define include directories
 INCLUDES = \
 		-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -31,10 +30,3 @@
 # define libraries
 wvcwopd_LDADD   =
 
-# define library directories
-wvcwopd_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wvcwopd_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
-
diff -Naur wview-5.21.7.orig/ftp/Makefile.am wview-5.21.7/ftp/Makefile.am
--- wview-5.21.7.orig/ftp/Makefile.am	2014-05-28 04:22:51.000000000 +0400
+++ wview-5.21.7/ftp/Makefile.am	2016-07-24 09:56:15.998458993 +0300
@@ -7,7 +7,6 @@
 
 # define include directories
 INCLUDES = 	-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-D_GNU_SOURCE
@@ -32,10 +31,4 @@
 # define libraries
 wviewftpd_LDADD   = -lssl -lcurl
 
-# define library directories
-wviewftpd_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewftpd_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/htmlgenerator/Makefile.am wview-5.21.7/htmlgenerator/Makefile.am
--- wview-5.21.7.orig/htmlgenerator/Makefile.am	2014-05-28 04:22:50.000000000 +0400
+++ wview-5.21.7/htmlgenerator/Makefile.am	2016-07-24 09:56:23.198056999 +0300
@@ -8,7 +8,6 @@
 # define include directories
 INCLUDES = \
 		-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -65,10 +64,4 @@
 # define libraries
 htmlgend_LDADD   =
 
-# define library directories
-htmlgend_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-htmlgend_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/http/Makefile.am wview-5.21.7/http/Makefile.am
--- wview-5.21.7.orig/http/Makefile.am	2014-05-28 04:22:51.000000000 +0400
+++ wview-5.21.7/http/Makefile.am	2016-07-24 09:55:57.385156970 +0300
@@ -8,9 +8,6 @@
 # define include directories
 INCLUDES = \
 		-I$(top_srcdir)/common \
-		-I$(prefix)/include \
-		-I/usr/include \
-		-I/usr/local/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -37,10 +34,4 @@
 # define libraries
 wvhttpd_LDADD     = -lssl -lcurl
 
-# define library directories
-wvhttpd_LDFLAGS   = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wvhttpd_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/procmon/Makefile.am wview-5.21.7/procmon/Makefile.am
--- wview-5.21.7.orig/procmon/Makefile.am	2014-05-28 04:22:51.000000000 +0400
+++ wview-5.21.7/procmon/Makefile.am	2016-07-24 09:55:35.186062999 +0300
@@ -8,7 +8,6 @@
 # define include directories
 INCLUDES = \
 		-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -32,10 +31,4 @@
 # define libraries
 wvpmond_LDADD   =
 
-# define library directories
-wvpmond_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wvpmond_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/ssh/Makefile.am wview-5.21.7/ssh/Makefile.am
--- wview-5.21.7.orig/ssh/Makefile.am	2014-05-28 04:22:50.000000000 +0400
+++ wview-5.21.7/ssh/Makefile.am	2016-07-24 09:56:07.882403015 +0300
@@ -7,7 +7,6 @@
 
 # define include directories
 INCLUDES = 	-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-D_GNU_SOURCE
@@ -32,10 +31,4 @@
 # define libraries
 wviewsshd_LDADD   =
 
-# define library directories
-wviewsshd_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewsshd_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/stations/Simulator/Makefile.am wview-5.21.7/stations/Simulator/Makefile.am
--- wview-5.21.7.orig/stations/Simulator/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/Simulator/Makefile.am	2016-07-24 09:58:01.779323024 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -51,10 +50,4 @@
 # define libraries
 wviewd_sim_LDADD       =
 
-# define library directories
-wviewd_sim_LDFLAGS     = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_sim_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/stations/TE923/Makefile.am wview-5.21.7/stations/TE923/Makefile.am
--- wview-5.21.7.orig/stations/TE923/Makefile.am	2014-05-17 05:06:19.000000000 +0400
+++ wview-5.21.7/stations/TE923/Makefile.am	2016-07-24 09:53:44.062528993 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
diff -Naur wview-5.21.7.orig/stations/TWI/Makefile.am wview-5.21.7/stations/TWI/Makefile.am
--- wview-5.21.7.orig/stations/TWI/Makefile.am	2014-05-28 04:22:48.000000000 +0400
+++ wview-5.21.7/stations/TWI/Makefile.am	2016-07-24 09:57:32.380631014 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -56,10 +55,4 @@
 # define libraries
 wviewd_twi_LDADD       =
 
-# define library directories
-wviewd_twi_LDFLAGS     = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_twi_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/stations/VantagePro/Makefile.am wview-5.21.7/stations/VantagePro/Makefile.am
--- wview-5.21.7.orig/stations/VantagePro/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/VantagePro/Makefile.am	2016-07-24 09:57:46.883878995 +0300
@@ -9,7 +9,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -55,10 +54,4 @@
 # define libraries
 wviewd_vpro_LDADD       =
 
-# define library directories
-wviewd_vpro_LDFLAGS     = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_vpro_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/stations/VantagePro/vpconfig/Makefile.am wview-5.21.7/stations/VantagePro/vpconfig/Makefile.am
--- wview-5.21.7.orig/stations/VantagePro/vpconfig/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/VantagePro/vpconfig/Makefile.am	2016-07-24 09:58:14.897878929 +0300
@@ -13,7 +13,6 @@
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
 		-I$(top_srcdir)/stations/VantagePro \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-D_VP_CONFIG_ONLY \
@@ -39,13 +38,6 @@
 # define libraries
 vpconfig_LDADD   =
 
-# define library directories
-vpconfig_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-vpconfig_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
-
 CLEANFILES = $(bin_SCRIPTS)
 
 vpinstall: $(srcdir)/vpinstall.sh
diff -Naur wview-5.21.7.orig/stations/Virtual/Makefile.am wview-5.21.7/stations/Virtual/Makefile.am
--- wview-5.21.7.orig/stations/Virtual/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/Virtual/Makefile.am	2016-07-24 09:57:23.079983074 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -57,10 +56,3 @@
 # define libraries
 wviewd_virtual_LDADD     =
 
-# define library directories
-wviewd_virtual_LDFLAGS   = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_virtual_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
-
diff -Naur wview-5.21.7.orig/stations/WH1080/Makefile.am wview-5.21.7/stations/WH1080/Makefile.am
--- wview-5.21.7.orig/stations/WH1080/Makefile.am	2014-05-17 05:05:54.000000000 +0400
+++ wview-5.21.7/stations/WH1080/Makefile.am	2016-07-24 09:53:03.814415004 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
diff -Naur wview-5.21.7.orig/stations/WMR918/Makefile.am wview-5.21.7/stations/WMR918/Makefile.am
--- wview-5.21.7.orig/stations/WMR918/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/WMR918/Makefile.am	2016-07-24 09:57:39.640258999 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -55,10 +54,3 @@
 # define libraries
 wviewd_wmr918_LDADD       =
 
-# define library directories
-wviewd_wmr918_LDFLAGS     = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_wmr918_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
-
diff -Naur wview-5.21.7.orig/stations/WMRUSB/Makefile.am wview-5.21.7/stations/WMRUSB/Makefile.am
--- wview-5.21.7.orig/stations/WMRUSB/Makefile.am	2014-05-17 05:04:25.000000000 +0400
+++ wview-5.21.7/stations/WMRUSB/Makefile.am	2016-07-24 09:53:14.863937037 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
diff -Naur wview-5.21.7.orig/stations/WS-2300/Makefile.am wview-5.21.7/stations/WS-2300/Makefile.am
--- wview-5.21.7.orig/stations/WS-2300/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/WS-2300/Makefile.am	2016-07-24 09:57:16.032461028 +0300
@@ -7,7 +7,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -55,10 +54,3 @@
 # define libraries
 wviewd_ws2300_LDADD       =
 
-# define library directories
-wviewd_ws2300_LDFLAGS     = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_ws2300_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
-
diff -Naur wview-5.21.7.orig/stations/WXT510/Makefile.am wview-5.21.7/stations/WXT510/Makefile.am
--- wview-5.21.7.orig/stations/WXT510/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/WXT510/Makefile.am	2016-07-24 09:57:54.319595000 +0300
@@ -10,7 +10,6 @@
 INCLUDES = \
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
-		-I$(prefix)/include \
 		-D_GNU_SOURCE \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -58,10 +57,4 @@
 # define libraries
 wviewd_wxt510_LDADD       =
 
-# define library directories
-wviewd_wxt510_LDFLAGS     = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wviewd_wxt510_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/stations/WXT510/wxt510config/Makefile.am wview-5.21.7/stations/WXT510/wxt510config/Makefile.am
--- wview-5.21.7.orig/stations/WXT510/wxt510config/Makefile.am	2014-05-28 04:22:49.000000000 +0400
+++ wview-5.21.7/stations/WXT510/wxt510config/Makefile.am	2016-07-24 09:58:23.950402989 +0300
@@ -8,7 +8,6 @@
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/common \
 		-I$(top_srcdir)/stations/WXT510 \
-		-I$(prefix)/include \
 		-D_WXT510_CONFIG_ONLY \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
@@ -37,10 +36,4 @@
 # define libraries
 wxt510config_LDADD   =
 
-# define library directories
-wxt510config_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wxt510config_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/utilities/archive-be2le/Makefile.am wview-5.21.7/utilities/archive-be2le/Makefile.am
--- wview-5.21.7.orig/utilities/archive-be2le/Makefile.am	2014-05-28 04:22:50.000000000 +0400
+++ wview-5.21.7/utilities/archive-be2le/Makefile.am	2016-07-24 09:56:39.118012998 +0300
@@ -8,7 +8,6 @@
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/VantagePro \
 		-I$(top_srcdir)/utilities \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-DBUILD_UTILITIES \
@@ -30,10 +29,4 @@
 # define libraries
 arc_be2le_LDADD   =
 
-# define library directories
-arc_be2le_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-arc_be2le_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/utilities/archive-le2be/Makefile.am wview-5.21.7/utilities/archive-le2be/Makefile.am
--- wview-5.21.7.orig/utilities/archive-le2be/Makefile.am	2014-05-28 04:22:50.000000000 +0400
+++ wview-5.21.7/utilities/archive-le2be/Makefile.am	2016-07-24 09:56:54.717809025 +0300
@@ -8,7 +8,6 @@
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/VantagePro \
 		-I$(top_srcdir)/utilities \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-DBUILD_UTILITIES \
@@ -30,10 +29,4 @@
 # define libraries
 arc_le2be_LDADD   =
 
-# define library directories
-arc_le2be_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-arc_le2be_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/utilities/hilowcreate/Makefile.am wview-5.21.7/utilities/hilowcreate/Makefile.am
--- wview-5.21.7.orig/utilities/hilowcreate/Makefile.am	2014-05-28 04:22:51.000000000 +0400
+++ wview-5.21.7/utilities/hilowcreate/Makefile.am	2016-07-24 09:57:08.364629000 +0300
@@ -6,7 +6,6 @@
 # define include directories
 INCLUDES = \
 		-I$(top_srcdir)/common \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-DBUILD_UTILITIES=1 \
@@ -27,10 +26,4 @@
 # define libraries
 hilowcreate_LDADD   =
 
-# define library directories
-hilowcreate_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-hilowcreate_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
diff -Naur wview-5.21.7.orig/utilities/sqlite2wlk/Makefile.am wview-5.21.7/utilities/sqlite2wlk/Makefile.am
--- wview-5.21.7.orig/utilities/sqlite2wlk/Makefile.am	2014-05-28 04:22:50.000000000 +0400
+++ wview-5.21.7/utilities/sqlite2wlk/Makefile.am	2016-07-24 09:56:47.922413003 +0300
@@ -8,7 +8,6 @@
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/VantagePro \
 		-I$(top_srcdir)/utilities \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-DBUILD_UTILITIES \
@@ -28,10 +27,3 @@
 # define libraries
 sqlite2wlk_LDADD   =
 
-# define library directories
-sqlite2wlk_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-sqlite2wlk_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
-
diff -Naur wview-5.21.7.orig/utilities/wlk2sqlite/Makefile.am wview-5.21.7/utilities/wlk2sqlite/Makefile.am
--- wview-5.21.7.orig/utilities/wlk2sqlite/Makefile.am	2014-05-28 04:22:51.000000000 +0400
+++ wview-5.21.7/utilities/wlk2sqlite/Makefile.am	2016-07-24 09:57:01.577236999 +0300
@@ -8,7 +8,6 @@
 		-I$(top_srcdir)/common \
 		-I$(top_srcdir)/stations/VantagePro \
 		-I$(top_srcdir)/utilities \
-		-I$(prefix)/include \
 		-DWV_CONFIG_DIR=\"$(sysconfdir)/wview\" \
 		-DWV_RUN_DIR=\"$(localstatedir)/wview\" \
 		-DBUILD_UTILITIES \
@@ -28,10 +27,4 @@
 # define libraries
 wlk2sqlite_LDADD   =
 
-# define library directories
-wlk2sqlite_LDFLAGS = -L$(prefix)/lib -L/usr/lib
-
-if CROSSCOMPILE
-wlk2sqlite_LDFLAGS += $(prefix)/lib/crt1.o $(prefix)/lib/crti.o $(prefix)/lib/crtn.o
-endif
 
