--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -6,7 +6,7 @@ project(6relayd C)
 
 set(CMAKE_SHARED_LIBRARY_LINK_C_FLAGS "")
 set(CMAKE_C_FLAGS "${CMAKE_C_FLAGS} -g -std=c99")
-add_definitions(-D_GNU_SOURCE -Wall -Werror -Wextra -Wno-extended-offsetof -pedantic)
+add_definitions(-D_GNU_SOURCE -Wall -Werror -Wextra -Wno-maybe-uninitialized -pedantic)
 
 add_executable(6relayd src/6relayd.c src/router.c src/dhcpv6.c src/ndp.c src/md5.c src/dhcpv6-ia.c)
 target_link_libraries(6relayd resolv)
