--- a/server/libgsm/Makefile
+++ b/server/libgsm/Makefile
@@ -53,7 +53,7 @@ CC		= gcc
 endif
 
 CC		+= -ansi -pedantic
-CCFLAGS 	= -c -O2 -DNeedFunctionPrototypes=1
+CCFLAGS 	= -c $(COPTS) -DNeedFunctionPrototypes=1
 
 LD 		= $(CC)
 
--- a/server/Makefile
+++ b/server/Makefile
@@ -47,7 +47,7 @@ ENABLE_X11MON=	0
 .PHONY : tools man
 
 
-CFLAGS= -Wall -D_REENTRANT -O2 -g
+CFLAGS= -Wall -D_REENTRANT $(COPTS)
 # -O3
 CC= gcc
 libjpeg= -ljpeg
