Description: assume unconfigured interface rather than exit
Author: KELEMEN Péter <fuji@debian.org>

Index: arpwatch/arpwatch.c
===================================================================
--- arpwatch.orig/arpwatch.c	2015-06-13 22:46:29.164588533 +0200
+++ arpwatch/arpwatch.c	2015-06-13 22:46:29.164588533 +0200
@@ -347,9 +347,10 @@
 
 		/* Determine network and netmask */
 		if (pcap_lookupnet(interface, &net, &netmask, errbuf) < 0) {
-			(void)fprintf(stderr, "%s: bad interface %s: %s\n",
-			    prog, interface, errbuf);
-			exit(1);
+			syslog(LOG_NOTICE, "bad interface %s: %s - assuming unconfigured interface",
+				interface, errbuf);
+			net = 0;
+			netmask = 0;
 		}
 
 		/* Drop into the background if not debugging */
