--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -10,19 +10,6 @@
 
 cmake_minimum_required(VERSION 2.8.11)
 
-# In-source builds are disabled.
-if ("${CMAKE_CURRENT_SOURCE_DIR}" STREQUAL "${CMAKE_CURRENT_BINARY_DIR}")
-    message(FATAL_ERROR
-        "CMake generation is not possible within the source directory!"
-        "\n Remove the CMakeCache.txt file and try again from another folder, e.g.:"
-        "\n "
-        "\n rm CMakeCache.txt"
-        "\n mkdir build"
-        "\n cd build"
-        "\n cmake .."
-    )
-endif()
-
 set(CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH};${CMAKE_CURRENT_SOURCE_DIR}/cmake")
 
 set(EXECUTABLE_OUTPUT_PATH "${CMAKE_CURRENT_BINARY_DIR}/bin")
