--- a/configure.in
+++ b/configure.in
@@ -142,27 +142,27 @@ REGEX_DIR=''
 REGEX_OBJS=''
 
 AC_ARG_ENABLE(pcre,
-[  --enable-pcre           use PCRE instead of GNU regex (default GNU)],
-[ use_pcre="$enableval" ],
-[ use_pcre="no" ])
+[  --enable-pcre           use PCRE instead of GNU regex (default GNU)])
+dnl [ use_pcre="$enableval" ],
+dnl [ use_pcre="no" ])
 
-if test use_pcre = yes; then
+dnl if test use_pcre = yes; then
   USE_PCRE="1"
   EXTRA_LIBS="$EXTRA_LIBS -lpcre"
-else
-  USE_PCRE="0"
+dnl else
+dnl  USE_PCRE="0"
 
-  AC_MSG_RESULT
-  AC_MSG_RESULT(Configuring GNU Regular Expression library ...)
-  AC_MSG_RESULT
+dnl  AC_MSG_RESULT
+dnl  AC_MSG_RESULT(Configuring GNU Regular Expression library ...)
+dnl  AC_MSG_RESULT
 
-  REGEX_DIR='regex-0.12'
-  REGEX_OBJS="$REGEX_DIR/regex.o"
+dnl  REGEX_DIR='regex-0.12'
+dnl  REGEX_OBJS="$REGEX_DIR/regex.o"
 
-  ( cd $REGEX_DIR && ./configure )
+dnl  ( cd $REGEX_DIR && ./configure )
 
-  EXTRA_INCLUDES="$EXTRA_INCLUDES -I$REGEX_DIR"
-fi
+dnl  EXTRA_INCLUDES="$EXTRA_INCLUDES -I$REGEX_DIR"
+dnl fi
 
 AC_SUBST(REGEX_DIR)
 AC_SUBST(REGEX_OBJS)
