--- a/configure.ac
+++ b/configure.ac
@@ -28,11 +28,6 @@
 if test x$PKGCONFIG = xno; then
   AC_MSG_ERROR([Required program pkg-config is missing or not in \$PATH])
 fi
-AC_CHECK_PROG([XSLTPROC],[xsltproc],[yes],[no])
-if test x$XSLTPROC != "xyes"; then
-    AC_MSG_ERROR([
-      xsltproc is required to build. It usually comes with libxml])
-fi
 AM_CONDITIONAL(HAVE_UINPUT, test -n "$HAVE_UINPUT" -o -e /dev/uinput)
 AC_CHECK_PROG([MAN2HTML],[man2html],[yes],[no])
 AM_CONDITIONAL(HAVE_MAN2HTML, test x$MAN2HTML = xyes)
@@ -40,7 +35,7 @@
 AM_CONDITIONAL(HAVE_DOXYGEN, test x$DOXYGEN = xyes)
 AC_PROG_LN_S
 LT_INIT([disable-static])
-AM_PATH_PYTHON([3.1],,)
+AM_PATH_PYTHON([2.7],,)
 AM_CONDITIONAL([HAVE_PYTHON], [test "$PYTHON" != ""])
 AC_CHECK_PROG([HAVE_PYTHON3],[python3],[yes],[no])
 LIBUSB_CONFIG="pkg-config libusb"
--- a/Makefile.am
+++ b/Makefile.am
@@ -25,7 +25,7 @@
     SYSTEMD_DIR         = systemd
 endif
 
-SUBDIRS                 = lib daemons tools plugins configs doc $(SYSTEMD_DIR)
+SUBDIRS                 = lib daemons tools plugins configs $(SYSTEMD_DIR)
 
 if INSTALL_ETC
 
@@ -44,8 +44,6 @@
 
 endif
 
-dist_doc_DATA           = VERSION
-
 pkgconfigdir            = $(libdir)/pkgconfig
 dist_pkgconfig_DATA     = lirc.pc lirc-driver.pc
 
