--- a/Makefile.am
+++ b/Makefile.am
@@ -19,7 +19,7 @@
     SYSTEMD_DIR         = systemd
 endif
 
-SUBDIRS                 = lib daemons tools doc $(SYSTEMD_DIR) plugins
+SUBDIRS                 = lib daemons plugins
 
 if INSTALL_ETC
 
@@ -44,7 +44,7 @@
 header_HEADERS          = paths.h drivers/irpipe/irpipe.h
 
 install-data-hook:
-	$(SED) -i -e '/^plugindir/s|/usr/lib|$(libdir)|' \
+	$(SED) -i -e '/^plugindir/s|/opt/lib|$(libdir)|' \
 	    $(DESTDIR)$(lirc_confdir)/lirc_options.conf
 
 uninstall-hook:
