--- a/lirc_options.conf
+++ b/lirc_options.conf
@@ -6,9 +6,9 @@
 nodaemon        = False
 driver          = default
 device          = /dev/lirc0
-output          = /var/run/lirc/lircd
-pidfile         = /var/run/lirc/lircd.pid
-plugindir       = /usr/lib/lirc/plugins
+output          = /opt/var/run/lirc/lircd
+pidfile         = /opt/var/run/lirc/lircd.pid
+plugindir       = /opt/lib/lirc/plugins
 permission      = 666
 allow-simulate  = No
 repeat-max      = 600
