--- a/configure.ac
+++ b/configure.ac
@@ -174,9 +174,6 @@ AH_TEMPLATE([HAVE_IOKIT_IOCFPLUGIN_H],
 
 AH_TEMPLATE([HAVE_LIBALSA], [Define if the ALSA library is installed])
 
-AH_TEMPLATE([HAVE_LIBPORTAUDIO],
-        [Define if the portaudio library is installed])
-
 AH_TEMPLATE([HAVE_LIBUSB], [Define if libusb is installed])
 
 AH_TEMPLATE([HAVE_LINUX_DEVINPUT],
@@ -253,10 +250,6 @@ else
   AM_CONDITIONAL([BUILD_FTDI],[false])
 fi
 
-dnl audio driver requires PortAudio library installed and some linker flags
-PKG_CHECK_MODULES([PORTAUDIO],[portaudio-2.0],[portaudio=yes],[portaudio=no])
-AM_CONDITIONAL(BUILD_LIBPORTAUDIO, [test x$portaudio = xyes])
-
 dnl audio_alsa driver requires ALSA library installed and some linker flags
 have_alsa=no
 AC_CHECK_HEADERS(alsa/asoundlib.h,[
@@ -556,7 +549,6 @@ host:                           $host
 host_os:                        $host_os
 forkpty:                        $forkpty
 LIBUSB_LIBS:                    $LIBUSB_LIBS
-PORTAUDIO_LIBS:                 $PORTAUDIO_LIBS
 
 Conditionals:
 ])
@@ -567,7 +559,6 @@ AX_REPORT_CONDITIONAL([BUILD_FTDI])
 AX_REPORT_CONDITIONAL([BUILD_HIDDEV])
 AX_REPORT_CONDITIONAL([BUILD_I2CUSER])
 AX_REPORT_CONDITIONAL([BUILD_LIBALSA])
-AX_REPORT_CONDITIONAL([BUILD_LIBPORTAUDIO])
 AX_REPORT_CONDITIONAL([BUILD_USB])
 AX_REPORT_CONDITIONAL([BUILD_XTOOLS])
 AX_REPORT_CONDITIONAL([HAVE_DOXYGEN])
