--- a/configure.in
+++ b/configure.in
@@ -42,15 +42,6 @@
 	"$MAKE" -C "$builddir/src" cleanall >/dev/null 2>/dev/null
 	AC_MSG_RESULT(done)
 fi
-
-# ===================================================================
-# Load feature configuration file and start logging features.
-# ===================================================================
-
-features="features.conf"
-test -f "$srcdir/$features" && . "$srcdir/$features"
-test -f "$builddir/$features" && . "$builddir/$features"
-echo "Feature summary:" > features.log
 
 # ===================================================================
 # Checks for programs.
