--- a/configure.ac
+++ b/configure.ac
@@ -21,7 +21,7 @@ AC_SUBST(PACKAGE)
 AC_SUBST(VERSION)
 AC_DEFINE_UNQUOTED(PACKAGE, "$PACKAGE", [Package version])
 AC_DEFINE_UNQUOTED(VERSION, "$VERSION", [Package version])
-AC_CONFIG_HEADERS(config.h)
+AC_CONFIG_HEADERS([config.h])
 
 AC_CHECK_PROG(ACLOCAL,[aclocal],[aclocal],[config/missing aclocal])
 AC_CHECK_PROG(AUTOCONF,[autoconf],[autoconf],[config/missing autoconf])
@@ -1430,7 +1430,8 @@ AC_CHECK_HEADERS(execinfo.h, HAVE_EXECIN
 
 ALL_LINGUAS="af be bg ca cs da de el es et fi fr gl hr hu id is it ja lt nl nb pl pt pt_BR ro ru sk sr sv tr uk"
 
-AM_GNU_GETTEXT
+AM_GNU_GETTEXT([external])
+AM_GNU_GETTEXT_VERSION(0.21)
 
 # iconv is always used
 LIBS="$LIBS $LIBICONV"
@@ -1816,7 +1817,9 @@ AC_LINK_IFELSE([AC_LANG_PROGRAM([[#inclu
 	#endif ]])],[cf_result=yes],[cf_result=no])
 
 if test "$cf_result" = "yes"; then
-	AC_DEFINE([HAVE_ALLOCA])
+	AC_DEFINE([HAVE_ALLOCA], [1],
+		[Define to 1 if you have 'alloca' after including <alloca.h>,
+		a header that may be supplied by this distribution.])
 	AC_DEFINE([_ALLOCA_H], [1], [Define as 1 if you are using Tiny C Compiler
 		with the GNU C Library,	and <alloca.h> of glibc would otherwise
 		override the alloca macro defined in <stddef.h> of TCC.
