--- a/src/main.c
+++ b/src/main.c
@@ -1059,15 +1059,6 @@
 
   printf("\n%s\n", version);
 
-#ifndef CYGWIN_HACKS
-  /* Don't allow eggdrop to run as root
-   * This check isn't useful under cygwin and has been
-   * reported to cause trouble in some situations.
-   */
-  if (((int) getuid() == 0) || ((int) geteuid() == 0))
-    fatal("ERROR: Eggdrop will not run as root!", 0);
-#endif
-
 #ifndef REPLACE_NOTIFIER
   init_threaddata(1);
 #endif
