--- a/xdg/Menu.py
+++ b/xdg/Menu.py
@@ -626,7 +626,7 @@ class XMLMenuBuilder(object):
             candidate = os.environ.get('XDG_MENU_PREFIX', '') + "applications.menu"
             filename = _get_menu_file_path(candidate)
         if not filename:
-            raise ParsingError('File not found', "/etc/xdg/menus/%s" % candidate)
+            raise ParsingError('File not found', "/opt/etc/xdg/menus/%s" % candidate)
         # check if it is a .menu file
         if not filename.endswith(".menu"):
             raise ParsingError('Not a .menu file', filename)
--- a/xdg/BaseDirectory.py
+++ b/xdg/BaseDirectory.py
@@ -32,13 +32,13 @@ xdg_data_home = os.environ.get('XDG_DATA
             os.path.join(_home, '.local', 'share')
 
 xdg_data_dirs = [xdg_data_home] + \
-    (os.environ.get('XDG_DATA_DIRS') or '/usr/local/share:/usr/share').split(':')
+    (os.environ.get('XDG_DATA_DIRS') or '/opt/share').split(':')
 
 xdg_config_home = os.environ.get('XDG_CONFIG_HOME') or \
             os.path.join(_home, '.config')
 
 xdg_config_dirs = [xdg_config_home] + \
-    (os.environ.get('XDG_CONFIG_DIRS') or '/etc/xdg').split(':')
+    (os.environ.get('XDG_CONFIG_DIRS') or '/opt/etc/xdg').split(':')
 
 xdg_cache_home = os.environ.get('XDG_CACHE_HOME') or \
             os.path.join(_home, '.cache')
@@ -130,7 +130,7 @@ def get_runtime_dir(strict=True):
             raise
         
         import getpass
-        fallback = '/tmp/pyxdg-runtime-dir-fallback-' + getpass.getuser()
+        fallback = '/opt/tmp/pyxdg-runtime-dir-fallback-' + getpass.getuser()
         create = False
 
         try:
