--- a/scheduler/auth.c
+++ b/scheduler/auth.c
@@ -2018,8 +2018,8 @@ cups_crypt(const char *pw,		/* I - Passw
     int			pwlen;		/* Length of password string */
     const char		*salt_end;	/* End of "salt" data for MD5 */
     char		*ptr;		/* Pointer into result string */
-    _cups_md5_state_t	state;		/* Primary MD5 state info */
-    _cups_md5_state_t	state2;		/* Secondary MD5 state info */
+    _cups_message_t	state;		/* Primary MD5 state info */
+    _cups_message_t	state2;		/* Secondary MD5 state info */
     unsigned char	digest[16];	/* MD5 digest result */
     static char		result[120];	/* Final password string */
 
