--- a/configure.ac
+++ b/configure.ac
@@ -775,9 +775,7 @@ then
   PKG_CHECK_MODULES([SSL],[libssl libcrypto],[LIBS="$LIBS $SSL_LIBS"],[
     AS_MESSAGE([SSL-check: pkg-config check failed, using traditional probe])
     LDFLAGS="$LDFLAGS -L$with_ssl/lib"
-    AC_LIB_LINKFLAGS([crypto])
-    AC_LIB_LINKFLAGS([ssl], [crypto])
-    LIBS="$LIBS $LIBSSL $LIBCRYPTO"
+    LIBS="$LIBS -lssl -lcrypto"
     dnl check if -ldl is needed
     AC_MSG_CHECKING([for additional library dependencies of SSL])
     found=0
@@ -801,13 +799,7 @@ else
   AC_MSG_WARN(Consider re-running configure --with-ssl.)
 fi
 
-AC_LIB_LINKFLAGS([crypto])
-AC_LIB_LINKFLAGS([ssl], [crypto])
-
-LIBS="$LIBS $LIBSSL $LIBCRYPTO"
-AC_MSG_NOTICE([LIBS:] "$LIBS")
-
-case "$LIBS" in *-lssl*|*libssl*)
+case "$LIBS" in *-lssl*)
 	AC_CHECK_DECLS([TLS1_3_VERSION],,
 		       AC_MSG_WARN([Your OpenSSL version is too old and does not support TLS v1.3. Upgrade.]),
 		       [#include <openssl/ssl.h>])
