--- a/Makefile.am
+++ b/Makefile.am
@@ -1,6 +1,6 @@
 # Makefile for fetchmail
 
-SUBDIRS=		. po
+SUBDIRS=		.
 AUTOMAKE_OPTIONS=	1.11 foreign no-dist-gzip dist-bzip2 dist-xz
 
 AM_CPPFLAGS=		-I$(srcdir)/libesmtp
@@ -172,7 +172,6 @@
 		m4/lock.m4 \
 		m4/longlong.m4 \
 		m4/nls.m4 \
-		m4/po.m4 \
 		m4/printf-posix.m4 \
 		m4/progtest.m4 \
 		m4/size_max.m4 \
@@ -231,8 +230,7 @@
 
 .PHONY: import-translations
 import-translations:
-	rsync -Lrtvz translationproject.org::tp/latest/fetchmail/  $(top_srcdir)/po
-	cd $(top_builddir)/po && $(MAKE) update-po
+	true
 
 SUFFIXES = .html .txt
 .txt.html:
