--- a/nzbget.conf
+++ b/nzbget.conf
@@ -2,10 +2,8 @@
 #
 # On POSIX put this file to one of the following locations:
 # ~/.nzbget
-# /etc/nzbget.conf
-# /usr/etc/nzbget.conf
-# /usr/local/etc/nzbget.conf
 # /opt/etc/nzbget.conf
+# /opt/share/nzbget/nzbget.conf
 #
 # On Windows put this file in program's directory.
 #
@@ -23,7 +21,7 @@
 #
 # On POSIX you can use "~" as alias for home directory (e.g. "~/downloads").
 # On Windows use absolute paths (e.g. "C:\Downloads").
-MainDir=~/downloads
+MainDir=/opt/downloads
 
 # Destination directory for downloaded files.
 #
@@ -93,12 +91,12 @@
 #
 # Set to empty value to disable the creating of the lock-file and the
 # check for another running instance (not recommended).
-LockFile=${MainDir}/nzbget.lock
+LockFile=/opt/var/lock/nzbget.lock
 
 # Where to store log file, if it needs to be created.
 #
 # NOTE: See also option <WriteLog>.
-LogFile=${DestDir}/nzbget.log
+LogFile=/opt/var/log/nzbget.log
 
 # Configuration file template.
 #
@@ -111,7 +109,7 @@
 # /usr/local/share/nzbget/nzbget.conf).
 #
 # Example: /usr/local/share/nzbget/nzbget.conf.
-ConfigTemplate=
+ConfigTemplate=/opt/share/nzbget/nzbget.conf.template
 
 # Required directories.
 #
